modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/privacy/UserAnalyticsSettings.kt

```diff
@@ -13,10 +13,23 @@ class UserAnalyticsSettings @Inject constructor(
     fun updateAnalyticsSetting(enabled: Boolean) {
         if (enabled) {
             settings.collectAnalytics.set(true, updateModifiedAt = true)
+            settings.collectAnalyticsThirdParty.set(true, updateModifiedAt = true)
             analyticsTracker.track(AnalyticsEvent.ANALYTICS_OPT_IN)
         } else {
             analyticsTracker.track(AnalyticsEvent.ANALYTICS_OPT_OUT)
             settings.collectAnalytics.set(false, updateModifiedAt = false)
+            settings.collectAnalyticsThirdParty.set(false, updateModifiedAt = false)
+            analyticsTracker.clearAllData()
+        }
+    }
+
+    fun updateAnalyticsThirdPartySetting(enabled: Boolean) {
+        if (enabled) {
+            settings.collectAnalyticsThirdParty.set(true, updateModifiedAt = true)
```

Comment:
We're updating the value for `collectAnalyticsThirdParty` and not using it anywhere right now, correct?

Reply:
Sorry I should have mentioned that I'm chaining the PRs. The next PR uses it https://github.com/Automattic/pocket-casts-android/pull/3830